### ISSUE TITLE
#401: guard against null returned from attempt()

### ISSUE THREAD

**Issue by uweschaefer** [factcast/factcast#402]
_From @uweschaefer on 2019-04-25 15:17:53 +0000 UTC_

_Copied from original issue: Mercateo/factcast#402_

**Comment by uweschaefer** [comment]
_From @codecov[bot] on 2019-04-25 15:26:21 +0000 UTC_

# [Codecov](https://codecov.io/gh/Mercateo/factcast/pull/402?src=pr&el=h1) Report
> Merging [#402](https://codecov.io/gh/Mercateo/factcast/pull/402?src=pr&el=desc) into [master](https://codecov.io/gh/Mercateo/factcast/commit/d71e1dd7d19403d1886862a218fbe4286a5fb521?src=pr&el=desc) will **increase** coverage by `0.02%`.
> The diff coverage is `100%`.


```diff
@@            Coverage Diff             @@
##           master     #402      +/-   ##
==========================================
+ Coverage    92.1%   92.12%   +0.02%     
==========================================
  Files          77       77              
  Lines        1671     1676       +5     
  Branches      149      150       +1     
==========================================
+ Hits         1539     1544       +5     
  Misses         73       73              
  Partials       59       59
```

**Comment by uweschaefer** [comment]
_From @uweschaefer on 2019-04-25 15:40:21 +0000 UTC_

closes #401
